cypress/e2e/tests/sticky/resize.spec.ts

```diff
@@ -190,7 +190,8 @@ context("Resize", () => {
   it("should shrink row height to min height", () => {
     // ✅
     const RESIZE_HEIGHT = -config.cellHeight;
-    
+
+    utils.scrollTo(0, 20);
```

Comment:
It needed to scroll down, the cursor was not tapping the line to resize @xdmaury